client/login/magic-login/index.jsx

```diff
@@ -93,7 +93,7 @@ class MagicLogin extends React.Component {
 				'magic-login': true,
 				'magic-login__request_link': ! magicLoginView,
 			} }>
-				<PageViewTracker path="/login" title="Login" />
+				<PageViewTracker path="/log-in/link" title="Login" />
```

Comment:
@jblz: you probably want to introduce a different path for each view of the magic login flow.